lib/rules/i18n-ellipsis.js

```diff
@@ -24,16 +25,13 @@ var rule = module.exports = function( context ) {
 			}
 
 			node.arguments.forEach( function( arg ) {
-				if ( 'Literal' !== arg.type || 'string' !== typeof arg.value ) {
-					return;
-				}
-
-				if ( -1 !== arg.value.indexOf( '...' ) ) {
+				let argumentString = getTextContentFromNode( arg );
```

Comment:
@deBhal : Yeah, if you look at the last build for this branch, you'll see that while green, the `npm test` step did produce an error:

https://circleci.com/gh/Automattic/eslint-plugin-wpcalypso/36